### ISSUE TITLE
Existing pages with react-pages-hierarchy web part broken after upgrade to 1.0.3

### ISSUE THREAD

**Issue by nmtoblum** [pnp/sp-dev-fx-webparts#2659]
### Disclaimer

Yes

### Sample

react-pages-hierarchy

### Author(s)

@bo_george @techienickb

### What happened?

We've been using the web part for quite some time in version 1.0.1 without big problems and tested now version 1.0.3. Existing pages don't seem to work after upgrade of the web part to 1.0.3.

The webparts that were previously working now look like this:
<img width="1086" alt="image" src="https://user-images.githubusercontent.com/57751598/165965806-2c79c536-e50e-41eb-a7ea-9c5a0cfff89a.png">

The console shows the following error:
<img width="1351" alt="image" src="https://user-images.githubusercontent.com/57751598/165965919-7a869a72-dd3d-48f7-b305-d618c5b06288.png">

The source for the error seems to be that the new version introduced the two web part properties "treeFrom" and "treeExpandTo":
<img width="382" alt="image" src="https://user-images.githubusercontent.com/57751598/165966274-731071cf-cb79-42a9-b932-a56211ceb1a8.png">

While the 1.0.1 didn't have these properties:
<img width="376" alt="image" src="https://user-images.githubusercontent.com/57751598/165966581-bcdb7f36-cbbf-4674-b41d-0b6af4b3f3b3.png">

While these properties seem technically only required for the new "Tree Hierarchy" view mode, if they're not present the existing "Ancestor" and "Children" modes break.

### Steps to reproduce

1. Install the web part in version 1.0.1
2. Add and configure a page with web part in "ancestor" and "children" mode so that it's working
3. Upgrade web part to 1.0.3
4. Web parts stay empty, see error in console

### Expected behavior

Existing pages that were created with the web part in version 1.0.1 shouldn't break when upgrading to 1.0.3.

BTW: The web part shows version 1.0.2 instead of 1.0.3 in the property pane.

### Target SharePoint environment

SharePoint Online

### Developer environment

macOS

### Browsers

- [ ] Internet Explorer
- [X] Microsoft Edge
- [X] Google Chrome
- [ ] FireFox
- [ ] Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Paste the results of SPFx doctor

m365 spfx doctor -o text (-o json doesn't work)
 
CLI for Microsoft 365 SharePoint Framework doctor
Verifying configuration of your system for working with the SharePoint Framework
 
✔ SharePoint Framework v1.14.0
✔ Node v14.19.0
✔ yo v4.3.0
✔ gulp v4.0.2
✔ bundled typescript used

**Comment by techienickb** [comment]
@nmtoblum can you try the version in my fork please?  I've added some null coalescing to the new properties for the tree view: https://github.com/techienickb/sp-dev-fx-webparts/tree/main/samples/react-pages-hierarchy

**Comment by nmtoblum** [comment]
Hi @techienickb,

thank you for taking care of the problem so quickly. Sorry that I didn't find time to test the fix earlier.

Unfortunately, I am having problems getting the version to run. The build runs without errors, but when I test the version on an affected page with "?debug=true&noredir=true&debugManifestsFile=https://localhost:4321/temp/manifests.js" (I can't deploy this version in the AppCatalog on the tenant) I get the following error message:
<img width="398" alt="image" src="https://user-images.githubusercontent.com/57751598/167170118-c911f896-470d-4b31-83e1-a817d208add7.png">

`Manifest not found for component id "0d910c1c-13b9-4e1c-9aa4-b008c5e42d7d" and version "16.14.0".`

I'm not sure what's the problem here. Do you have an idea?

Greetings
Tobias

**Comment by nmtoblum** [comment]
Hi @techienickb,

sorry that it took me so long to try it out, but now I finally found time.

I suspected that the problem had something to do with the rush compiler but couldn't figure it out. :-)

I can confirm that this version works in my tests, even with pages that were created with the old version that don't have the additional fields populated in the web part settings.

Thank you for getting this fixed!

Greetings
Tobias
